tests/integration/app_3001_views/template_engine_null_data_resource_test.ts

```diff
@@ -12,6 +12,7 @@ function getExpected() {
 }
 
 const server = new Drash.Http.Server({
+  directory: "./",
```

Comment:
worried about this, does `directory: "."` not work anymore? And i'm worried because any drash servers will break from this PR as it'll be expecting `./`, whereas all drash apps **currently** use `.`

Reply:
9c57d68

Reply:
9c57d68